PaddleCV/object_detection/ppdet/utils/coco_eval.py

```diff
@@ -184,9 +184,9 @@ def mask2out(results, clsid2catid, resolution, thresh_binarize=0.5):
                     resized_mask > thresh_binarize, dtype=np.uint8)
                 im_mask = np.zeros((im_h, im_w), dtype=np.uint8)
 
-                x0 = max(xmin, 0)
+                x0 = min(max(xmin, 0), im_w)
                 x1 = min(xmax + 1, im_w)
-                y0 = max(ymin, 0)
+                y0 = min(max(ymin, 0), im_h)
```

Comment:
also need to check x1, y1

x1 = max(min(xmax + 1, im_w), 0)
y1 = max(min(ymax + 1, im_h), 0)

Reply:
Done